roottest/root/io/filemerger/CMakeLists.txt

```diff
@@ -1,372 +1,407 @@
 ROOTTEST_ADD_TEST(execKeyOrder
-	          MACRO execKeyOrder.C
-		  OUTREF execKeyOrder.ref)
-
+	               MACRO execKeyOrder.C
```

Comment:
Both the original and the new version contains tab characters, can you update to use space?

Reply:
Done.